src/commands/RunCommand.js

```diff
@@ -12,7 +13,20 @@ export default class RunCommand extends Command {
       return;
     }
 
-    this.packagesWithScript = this.packages
+    let packagesToRunCommandIn = this.packages;
+
+    if (this.flags.restrictTo && `${this.flags.restrictTo}`.length > 0) {
```

Comment:
just saw your comment @thejameskyle - if you want to not run anything in case an empty restriction is passed, then I think I need to add another check. Right now it will behave as if no restriction was given when passing an empty one (which I think might feel more natural).